docusaurus/docs/iOS/guides/working-with-user.md

```diff
@@ -118,21 +118,29 @@ chatClient.currentUserController().reloadUserIfNeeded()
 ```
 to make sure `ChatClient` calls your `tokenProvider` closure and acquires a new token for the newly logged-in user.
 
-You don't need to "logout" in the common sense. Deallocating `ChatClient` instances for the currently logged-in user means the user will disconnect. If you want to keep your "ChatClient" instance (eg you're using a singleton for it) you can call:
+If you're using static tokens for developing quick prototypes and want to test login/logout, you can assign a new `tokenProvider` to the `ChatClient` and then call `reloadUserIfNeeded` to "login" using the new token:
+```swift
+chatClient.tokenProvider = .static(newToken)
+chatClient.currentUserController().reloadUserIfNeeded()
+```
+
+## Disconnect & Logout
```

Comment:
Move this section to the Getting Started guide, after the "Connect User" section 👍